cli/shared/src/main/scala/com/lightbend/rp/reactivecli/docker/DockerEngine.scala

```diff
@@ -53,7 +54,7 @@ object DockerEngine extends LazyLogging {
       case Some(host) if host.startsWith("tcp://") =>
         val verify = env.get("DOCKER_TLS_VERIFY").contains("1")
         val protocol = if (verify) "https" else "http"
-        val url = s"$protocol://${host.replaceFirst("tcp://", "")}/images/$uri/json"
+        val url = Platform.encodeURI(s"$protocol://${host.replaceFirst("tcp://", "")}/images/$uri/json")
```

Comment:
Could you move this to `package.scala` (i.e. adding a method `encodeURI` that simply calls `Platform.encodeURI`) in the `http` package (see `files` for similar reference).

My intent is to wrap the platform-specific code in methods inside the `shared` project so that 1) Usage of `Platform` is fairly contained, and 2) IDE stuff continues to work.

Reply:
Ok, moved encodeURI and parseAuthHeader into http package. One small issue - there can't be http.scala and Http.scala files in the same dir. So I put http.scala outside.